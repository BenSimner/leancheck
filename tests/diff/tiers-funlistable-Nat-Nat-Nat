tiers :: [Nat->Nat->Nat]  =
  [[]]

map length (tiers :: [[ Nat->Nat->Nat ]])  =  [0]

length (list :: [ Nat->Nat->Nat ])  =  0
