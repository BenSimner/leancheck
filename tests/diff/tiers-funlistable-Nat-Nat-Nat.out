tiers :: [Nat->Nat->Nat]  =
  [[]]

map length (tiers :: [[ Nat->Nat->Nat ]])  =  [0]

length (list :: [ Nat->Nat->Nat ])  =  0

allUnique (list :: [ Nat->Nat->Nat ])  =  True
